### PR TITLE
Update design

### PR DESCRIPTION
**Pull request by Kriesse**
Adjusted header, button and input styles.
Updated to us the latest versions of dat-design and dat-icons.

Preview: 
image

### CODE REVIEW

File: public/img/delete.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 240 240"><title>delete</title><path d="M58 189.89A26.14 26.14 0 0 0 84.11 216h71.78A26.14 26.14 0 0 0 182 189.89V72H58zM182.82 34.4a134.54 134.54 0 0 0-14.39-3.19c-4.61-.87-9.37-1.76-14-2.9a22.86 22.86 0 0 1-8.3-4.41 14.64 14.64 0 0 0-9.94-3.9h-32a14.64 14.64 0 0 0-9.94 3.91 22.86 22.86 0 0 1-8.3 4.41c-4.66 1.15-9.52 2.05-14.22 2.93a139.12 139.12 0 0 0-14.47 3.17C49.79 36.72 46 40.66 46 46.11V60h148V46.11c0-5.44-3.76-9.38-11.18-11.71z"/></svg>
```
Comment (yoshuawuyts) [inline public/img/delete.svg]: should these files still be part of the repo now that we've moved to `dat-icons`?

Comment (Kriesse) [inline public/img/delete.svg]: ah right! removed them.

**Comment by karissa** [comment]
LGTM!

**Comment by yoshuawuyts** [comment]
tested locally, looking good!

**Pull request merged**